src/server/master_node_attr_def.xml

```diff
@@ -494,6 +494,24 @@
 	<ECL>NULL_VERIFY_VALUE_FUNC</ECL>
 	</member_verify_function>
    </attributes>
+   <attributes>
+   /* ND_ATR_MaintJobs */
+        <member_name><both>ATTR_NODE_MaintJobs</both></member_name>             <!-- maintenance_jobs -->
+        <member_at_decode>decode_arst </member_at_decode>
+        <member_at_encode>encode_arst </member_at_encode>
+        <member_at_set>set_arst </member_at_set>
+        <member_at_comp>comp_arst </member_at_comp>
+        <member_at_free>free_arst </member_at_free>
+        <member_at_action>NULL_FUNC </member_at_action>
```

Comment:
I'm sure this resulted from cutting and pasting, but why are there spaces embedded in these markups? I suggest you file a ticket to remove them. It need not be done as part of this pull request.